### DISCUSSION TITLE
add 'remember me' feature to login

### DISCUSSION THREAD

**Discussion by benanhalt** [specify/specify7#5512] [Feature Requests]
see:
set_expiry function in
https://docs.djangoproject.com/en/3.2/topics/http/sessions/#django.contrib.sessions.backends.base.SessionBase.set_expiry

and

https://djangosnippets.org/snippets/1881/

should have master setting to enable.

**Comment by maxpatiiuk** [comment]
See also https://www.digitala11y.com/understanding-sc-2-2-1-timing-adjustable/

**Comment by maxpatiiuk** [comment]
New release partially negated the need for this by adding SSO support
